skyportal/utils/offset.py

```diff
@@ -813,12 +848,20 @@ def get_finding_chart(
                 hdu.data, source_image_parameters[image_source]["smooth"] / pixscale
             )
 
-        percents = np.nanpercentile(im.flatten(), [10, 99.0])
-        vmin = percents[0]
-        vmax = percents[1]
-        norm = ImageNormalize(im, vmin=vmin, vmax=vmax, interval=ZScaleInterval())
-        watermark = source_image_parameters[image_source]["str"]
-    else:
+        cent = int(npixels / 2)
+        width = int(0.05 * npixels)
+        test_slice = slice(cent - width, cent + width)
+        all_nans = np.isnan(im[test_slice, test_slice].flatten()).all()
+        all_zeros = (im[test_slice, test_slice].flatten() == 0).all()
+        if not (all_zeros or all_nans):
+            percents = np.nanpercentile(im.flatten(), [10, 99.0])
+            vmin = percents[0]
+            vmax = percents[1]
+            norm = ImageNormalize(im, vmin=vmin, vmax=vmax, interval=ZScaleInterval())
+            watermark = source_image_parameters[image_source]["str"]
+            fallback = False
```

Comment:
We got a good image that has normal fluxes expected around the source position. Do not fallback.

Reply:
Maybe do something like
```python
interval = ZScaleInterval(
    nsamples=im.shape[0] * im.shape[1],
    contrast=0.045,
    krej=2.5
)
vmin, vmax = interval.get_limits(im)
```
and then make `contrast` and `krej` configurable in the call?

Reply:
Done.

Reply:
Thanks!